js_modules/dagit/packages/core/src/gantt/GanttStatusPanel.tsx

```diff
@@ -154,7 +154,7 @@ const StepItem: React.FunctionComponent<{
           position="bottom"
           content={'Unknown step state. Run completed without step execution completion.'}
         >
-          {'?'}
+          <div style={{width: 12, textAlign: 'center'}}>{'?'}</div>
```

Comment:
Is there an actual `IconWIP` we could use for this?

Reply:
Ahh I think so, if nothing else we could add an icon or abstract this out a bit, will see!